sdks/python/hatchet_sdk/hatchet.py

```diff
@@ -294,8 +293,8 @@ def task(
         sticky: StickyStrategy | None = None,
         default_priority: int = 1,
         concurrency: ConcurrencyExpression | list[ConcurrencyExpression] | None = None,
-        schedule_timeout: Duration = DEFAULT_SCHEDULE_TIMEOUT,
-        execution_timeout: Duration = DEFAULT_EXECUTION_TIMEOUT,
+        schedule_timeout: Duration = timedelta(minutes=5),
```

Comment:
are we sure we dont want a constant here?

Reply:
we do :( but it messes up the docs - they end up showing `DEFAULT_XYZ_TIMEOUT` instead of the actual value. I wrote the tests for this to try to enforce what the constant would give us 🤷

Reply:
<img width="811" alt="Screenshot 2025-05-06 at 5 20 06 PM" src="https://github.com/user-attachments/assets/4a24457e-766a-4f24-b485-9b65a0017039" />

Reply:
gotcha, ok good enough